tasks/main.yml

```diff
@@ -32,7 +43,7 @@
     become: True
     template:
       src: ldap.conf.j2
-      dest: /etc/ldap/ldap.conf
+      dest: "{{ auth_client_ca | dirname }}/ldap.conf"
```

Comment:
This assumes the CA resides in the same directory as the config file. However, it's backward compatible